snuba/web/rpc/v1/resolvers/R_eap_items/storage_routing/routing_strategies/linear_bytes_scanned_storage_routing.py

```diff
@@ -288,16 +287,6 @@ def _emit_estimation_error_info(
     def _output_metrics(self, routing_context: RoutingContext) -> None:
         assert routing_context.query_result
         target_tier = routing_context.query_settings.get_sampling_tier()
-        query_duration_ms = self._get_query_duration_ms(routing_context.query_result)
-        if (
-            query_duration_ms
-            >= self._get_time_budget_ms() - 0.02 * self._get_time_budget_ms()
-        ):
-            self.metrics.increment(
-                "timeout_overflow_mode_was_hit",
```

Comment:
I'm assuming we don't need this since we will know if it's hit via the exception